.idea/codeStyleSettings.xml

```diff
@@ -1,699 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Oh lol, you deleted the code style. Can you try resetting your HEAD and making the doc fix again?